components/entitlement/org.wso2.carbon.identity.entitlement.ui/src/main/resources/web/entitlement/select-attribute.jsp

```diff
@@ -266,20 +267,20 @@
                 createInputs(paths[i].name);
             }
         }
-        document.attributeValueForm.action = "<%=returnPage%>.jsp?category="
-                + '<%=category%>' +"&ruleId=" + '<%=ruleId%>' ;
+        document.attributeValueForm.action = "<%=Encode.forUriComponent(returnPage)%>.jsp?category="
```

Comment:
returnPage is not included in a query string. Encode for javascript block instead